### PR TITLE
Added Tooling and scripts for oscd

### PR DESCRIPTION
**Pull request by pascalwilbrink**

### CODE REVIEW

File: tools/oscd/src/generators/component/files/src/__selector__.ts.template [hunk 1]
```diff
@@ -9,14 +9,19 @@ import { OscdComponent } from '@openscd/core';
...
+ * @tagname <%= selector %>
  */
 export class <%= className %> extends OscdComponent {
-
   static styles: CSSResultGroup = styles;
 
-  render(): TemplateResult {
-    return html`<span><%= selector %> works!</span>`
+  static get scopedElements() {
+    return {};
```
Comment (juancho0202) [inline tools/oscd/src/generators/component/files/src/__selector__.ts.template:21]: what is the purpose of this empty object being returned for new components?

**Review by juancho0202 — CHANGES_REQUESTED** [review]
One minor change and an open question 👍

File: tools/oscd/src/generators/component/files/src/__selector__.ts.template (hunk 1 above)
Comment (pascalwilbrink) [reply tools/oscd/src/generators/component/files/src/__selector__.ts.template:21]: There is no purpose, it's only put in to remind devs that scopedElements are possible.

Comment (pascalwilbrink) [reply tools/oscd/src/generators/component/files/src/__selector__.ts.template:21]: I can remove it, but we have to keep in mind that this is possible. 
Scoped Registries are actively being worked on ATM.

Comment (juancho0202) [reply tools/oscd/src/generators/component/files/src/__selector__.ts.template:21]: Oh ok, just wasn't familiar with the concept.
https://open-wc.org/docs/development/scoped-elements/ you mean this thing, right?
What are we planning on using that for?

**Review by juancho0202 — APPROVED** [review]

**Pull request merged**